packages/SwingSet/src/devices/mailbox/mailbox.js

```diff
@@ -71,19 +71,42 @@ import { Nat } from '@endo/nat';
 // replace it with one that tracks which parts of the state have been
 // modified, to build more efficient Merkle proofs.
 
+/**
+ * @typedef {object} Mailbox
+ * @property {bigint} ack
+ * @property {Map<bigint, unknown>} outbox
+ */
+
+/**
+ * @typedef {object} MailboxExport
+ * @property {number} ack
+ * @property {Array<number, unknown>} outbox
```

Comment:
That's not valid. I think you want:
```suggestion
 * @property {Array<number, unknown>} outbox
```
I think it was allowed because JSDoc isn't validated as strictly. Consider putting these in a types.ts.